projects/addon-doc/src/components/language-switcher/language-switcher.component.html

```diff
@@ -0,0 +1,26 @@
+<tui-select
+    tuiTextfieldSize="m"
+    [valueContent]="content"
+    [formControl]="language"
+>
+    <ng-content></ng-content>
+    <ng-template tuiDataList>
```

Comment:
We write this as structural directive everywhere, let's keep it consistent.

Reply:
fixed